packages/salesforcedx-vscode-apex/src/ApexErrorHandler.ts

```diff
@@ -0,0 +1,66 @@
+import { EventEmitter } from 'events';
+import {
+  CloseAction,
+  ErrorAction,
+  ErrorHandler,
+  LanguageClient,
+  LanguageClientOptions,
+  Message,
+  ServerOptions
+} from 'vscode-languageclient';
+
+export class ApexErrorHandler extends EventEmitter implements ErrorHandler {
```

Comment:
@lukeis one concern I had about using nodejs EventEmitter is it the correct base class to use in a vscode extension. Given the extensions use events/notifications, might there be a better choice.